CorsixTH/Src/th_gfx_common.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2009-2013 Peter "Corsix" Cawley and Edvin "Lego3" Linge
```

Comment:
If this is a new file in its entirety (can't see if it's sourced from somewhere else) Corsix and Edvin are not the people for this line. It is yourself and perhaps driverpt.

Ideally also add yourself to the license.txt file

Reply:
Thanks, something like I've done in the latest commit?